setup.cfg

```diff
@@ -1,6 +1,14 @@
 [flake8]
-max-line-length = 99
-
+max-line-length = 99,
+exclude = setup.py,asv/commands,asv/extern,asv/plugins,asv/template/benchmarks,
```

Comment:
This looks great, thanks.

Only thing I'd do, is to have one file per line. It looks better the way you did it, as it's not super long, but with one line per file PRs will be easier to review when we keep fixing the pending files. And also, conflicts will be easier to solve, if files are fixed in parallel.

Reply:
Yes, I agree. Let me fix this